src/NuGetPackageVerifier/third-party.json

```diff
@@ -41,11 +41,11 @@
     "Serilog.Sinks.File": [ "3.2.0" ],
     "Serilog.Sinks.PeriodicBatching": [ "2.1.0" ],
     "Serilog.Sinks.RollingFile": [ "3.3.0" ],
-    "SQLitePCLRaw.bundle_green": [ "1.1.3", "1.1.5" ],
-    "SQLitePCLRaw.core": [ "1.1.3", "1.1.5" ],
-    "SQLitePCLRaw.lib.e_sqlite3.v110_xp": [ "1.1.3", "1.1.5" ],
-    "SQLitePCLRaw.lib.e_sqlite3.linux": [ "1.1.3", "1.1.5" ],
-    "SQLitePCLRaw.lib.e_sqlite3.osx": [ "1.1.3", "1.1.5" ],
+    "SQLitePCLRaw.bundle_green": [ "1.1.3", "1.1.5", "1.1.6" ],
+    "SQLitePCLRaw.core": [ "1.1.3", "1.1.5", "1.1.6" ],
+    "SQLitePCLRaw.lib.e_sqlite3.v110_xp": [ "1.1.3", "1.1.5", "1.1.6" ],
+    "SQLitePCLRaw.lib.e_sqlite3.linux": [ "1.1.3", "1.1.5", "1.1.6" ],
+    "SQLitePCLRaw.lib.e_sqlite3.osx": [ "1.1.3", "1.1.5", "1.1.6" ],
```

Comment:
If the tool versioned with our code, I would delete old versions. But in theory old builds of the code might be built with new versions of the tool. Ideally this config data won't ship in the tool. #266